stabilize.py

```diff
@@ -1,6 +1,16 @@
 #! /usr/bin/env python
 # coding: utf-8
 
+"""
+stabilize image using sparse optical flow.
+dump numpy file that stores 3d(x, y, z) correction direction.
+
+Please configure in [STABILIZE] section of config/config.ini file.
+    ANGLE_THRESH: If an one frame has flow angle varience under ANGLE_THRESH, it is not detect feature in this frame. 
```

Comment:
If the variance of the flows in a frame is under ANGLE_THRESH, features in the frame are thought to be undetected.